pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -30,6 +30,11 @@ import (
 
 	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	autoscalingv1 "agones.dev/agones/pkg/apis/autoscaling/v1"
+	listeragonesv1 "agones.dev/agones/pkg/client/listers/agones/v1"
+	"agones.dev/agones/pkg/fleets"
+	"agones.dev/agones/pkg/gameservers"
+	gsSets "agones.dev/agones/pkg/gameserversets"
```

Comment:
Can we use `gameserversets` or `gssets` since package names are all lower case, and otherwise I end up looking for a `gsSets` object 😄 

```suggestion
	"agones.dev/agones/pkg/gameserversets"
```

Reply:
Ah, didn't realize they were all lowercase. Made the change.